src/main/java/mate.academy/Main.java

```diff
@@ -4,13 +4,15 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
 
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService =
+                (ProductService) injector.getInstance(ProductServiceImpl.class);
```

Comment:
Direct casting to 'ProductService' can lead to a 'ClassCastException' if the instance returned by 'getInstance' is not of 'ProductService' type. It's better to ensure that the injector returns the correct type or handle the potential exception.